pkgs/development/python-modules/panda3d/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchurl
+, python
+, isPy37
+, isPy38
+, isPy39
+, autoPatchelfHook
+, libXext
+, libGL
+, gdk-pixbuf
+, cairo
+, pango }:
+
+let pyVerNoDot = builtins.replaceStrings [ "." ] [ "" ] python.pythonVersion;
+    version = "1.10.10";
+    srcs = import ./binary-hashes.nix version;
+    unsupported = throw "Unsupported system";
+in buildPythonPackage rec {
+  pname = "panda3d";
+
+  inherit version;
+
+  format = "wheel";
+
+  src = fetchurl srcs."${stdenv.system}-${pyVerNoDot}" or unsupported;
+
+  disabled = !(isPy37 || isPy38 || isPy39);
+
+  # NOTE(breakds): Panda3d comes with a set of binary tools. There is one tool
+  # called `pstats` that I cannot find proper libraries to patchelf it. Since it
+  # does not affect using panda3d as a library, remove it before applying
+  # patchelf.
+  postInstall = ''
+   rm $out/lib/python${python.pythonVersion}/site-packages/panda3d_tools/pstats
+  '';
```

Comment:
the missing libraries are in `atk`, and `gtk2` respectively.

Since it's GTK it might additionally need wrapping with `wrapGAppsHook`

Reply:
Thanks! Let me try this.